shared/api/instance.go

```diff
@@ -374,3 +378,80 @@ type InstanceSource struct {
 	// API extension: instance_allow_inconsistent_copy
 	AllowInconsistent bool `json:"allow_inconsistent" yaml:"allow_inconsistent"`
 }
+
+// InstanceEFI represents the EFI vars of a LXD virtual machine instance.
+//
+// swagger:model
+//
+// API extension: instances_efi_vars.
+type InstanceEFI struct {
+	// EFI variables map
+	// Hashmap key format is <efi-variable-name>-<UUID>
+	// Example: AmdAcpiVar-79941ecd-ed36-49d0-8124-e4c31ac75cd4
+	Variables map[string]*InstanceEFIVariable `json:"variables" yaml:"variables"`
```

Comment:
Is there a reason for using a pointer to InstanceEFIVariable as the map entry values?

Reply:
No, I think we can go just with `map[string]InstanceEFIVariable` then.

Reply:
Fixed.